unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/WearableController.cs

```diff
@@ -59,7 +64,7 @@ public virtual void Load(string bodyShapeId, Transform parent, Action<WearableCo
 
         loader.settings.forceNewInstance = false;
         loader.settings.initialLocalPosition = Vector3.up * 0.75f;
-        loader.settings.cachingFlags = MaterialCachingHelper.Mode.CACHE_SHADERS;
+        loader.settings.cachingFlags = MaterialCachingHelper.Mode.CACHE_EVERYTHING;
```

Comment:
We can cache everything now because we don't modify the loaded materials as we did before.